lib/rails_response_dumper/runner.rb

```diff
@@ -90,7 +93,7 @@ def run_dumps
               response: {
                 status: response.status,
                 status_text: response.status_message,
-                headers: response.headers,
+                headers: response_headers,
```

Comment:
When `--exclude-response-headers` is passed, I expect this `headers` key to not exist at all.

Reply:
I was debating this, thinking it might be nice to keep the same shape, but I'm fine with removing this from the output.

Done in @nickdufresne
[Do not output headers property at all](https://github.com/Pioneer-Valley-Books/rails-response-dumper/pull/68/commits/d20490d75661034698b9b23e844d008edf3e390e)